index.html

```diff
@@ -23,7 +23,20 @@
       <button class="gameSpace" id="8"></button>
       <button class="gameSpace" id="9"></button>
     </div>
+    <br />
     <div id="text"></div>
+    <br />
+    <form>
+      <label for="P1Name">Player X:</label>
```

Comment:
I don't think you wanted any of this player name stuff to be in this Restart Button PR.
You should remove it before you merge